src/libp2p_transport.erl

```diff
@@ -59,8 +59,8 @@ for_addr(TID, Addr) ->
 %%--------------------------------------------------------------------
 %% @doc
 %% Addresses are sorted by priority ranking from 1 to 5 (1 highest and 5 lowest priority)
-%% 1 = NON rfc1918 IP (public IPs), 2 = P2P circuit address (relay), 3 = P2P address,
-%% 4 = rfc1918 IPs (private/local IPs), 5 = Proxy transport
+%% 1 = Non-Private IP (public IPs), 2 = P2P circuit address (relay), 3 = P2P address,
+%% 4 = Public IPs (shared/reserved/private/local IPs), 5 = Proxy transport
```

Comment:
Don't you mean *Private* IPs instead of `Public IPs` here? 

Nit: Actually these things technically aren't all private. Some are multicast or reserved. Personally, I think using the term bogon in code will lead future devs to look up what that means and understand.  This "private vs bogon" comment applies to this whole PR.

Reply:
Good catch on the Public when I meant private. My use of private_ip was to make it a little more obvious but fair point on it not just being private IPs. I will play with some other names for the function.